docker/scripts/install_common_deps.sh

```diff
@@ -17,16 +17,17 @@ sudo apt-get install --no-install-recommends -y \
   cppcheck \
   curl \
   git \
-  g++-8 \
-  pkg-config \
+  g++ \
+  lcov \
+  pkg-config
 
 sudo apt-get install --no-install-recommends -y \
-  clang-tidy-6.0 \
-  python-yaml \
-  libclang-6.0-dev
+  clang-tidy \
+  python3-yaml
 
 sudo apt-get install --no-install-recommends -y \
   libbenchmark-dev \
-  libbenchmark1
+  libbenchmark1.8.3
```

Comment:
I think we can get rid of `libbenchmark1.8.3`. It should be automatically installed with `libbenchmark-dev`

Reply:
`libbenchmark` is an optional dependency. There are tests that use it, but they're disabled if `libbenchmark` is not installed https://github.com/gazebosim/gz-sim/blob/c298007066eb3a0a4b260884fe9a1148903e7bc3/test/benchmark/CMakeLists.txt#L4

So I think we should keep it.

Reply:
Reverted, commend added